app/views/shared/_spam_honeypot.html.erb

```diff
@@ -0,0 +1,5 @@
+<%# Honeypot field intended to reduce spam %>
+<div class="visually-hidden" aria-hidden="true">
+  <label for="val">Ignore if you're not a robot</label>
+  <input type="text" tabindex="-1" autocomplete="off" id="val" name="<%= "#{form_name}" %>[val]"/>
```

Comment:
Ah, that makes sense. But if this had `display: none` like it had before, it wouldn't be necessary, right?
If "inline styling [is] more easily detected by spambots", would it be worth to use the `hidden` class instead. That uses `display: none` but not inline. Then you wouldn't need to use `aria-hidden` and `tabindex`.
Or do spambots look for the applied style from external CSS files as well, not just the inline style?